webhook/handlers/UserController.js

```diff
@@ -0,0 +1,177 @@
+    let patients = await db.patientWA.findAll({where: {User_id: user.id}}).catch(err => {return err;});
+    
+    if (patients.length === 0) return "You need to register a patient first. Would you like to register a new patient?";
+    for(let i=0;i< patients.length;i++) {
+      let patient = patients[i];
+      fulfillmentText = fulfillmentText + "\n\nPatient " + (i+1).toString() + ": " + 
+        "\nfirst_name: " + patient.first_name +
+        "\nlast_name: " + patient.last_name;
+      let pInfo = await db.patientInfoWA.findOne({where: {patientWAid: patient.id}}).catch(err => {return err;});
+      fulfillmentText += "\ndob: " + pInfo.dob +
+        "\ngender: " + pInfo.sex +
+        "\Contact: " + pInfo.phone_number +
+        "\nheight: " + pInfo.height_cm + " cm" + 
+        "\nweight: " + pInfo.weight_kg + " kg" +
+        "\nblood_type: " + pInfo.blood_type;
+    };
+    return fulfillmentText;
+  }
+
+  update(agent) {
+    let cNum = agent.parameters["telephone"];
+    //if(db.userWA.findAll({where: {phone_number: cNum}}).length > 0) return "Phone Number already Exists";
+    db.patientWA.update(agent.parameters, {where: {phone_number: cNum}})
+      .then(() => {return "Patient Updated";})
+      .catch(err => {return err;});
+  }
+  
+  remove(agent) {
+    let cNum = agent.parameters["telephone"];
+    db.patientWA.destroy({
+      where: { telephone: cNum }
+    }).then(() => {return "Patient Deleted";})
+    .catch(err => {return err;});
+  }
+
+  async show(agent, body) {
+    let user = await db.userWA.findOne({where: {wa_phone_number: body.From}});
+    let fulfillmentText = "These are the details of your Patient: ";
+    db.patientWA.findAll({where: { user_id : user.id, 
+                                  first_name: agent.parameters["patient-given-name"]}
+                                }).then(patients => {
+                                  if(patients.length == 1){
+                                    let patwa = patients[0];
+                                    let patient = await db.patientInfoWA.findOne({where: {patientWAid: patwa.id}});
+                                    fulfillmentText = fulfillmentText +  
+                                                            "\nfirst_name: " + patwa.first_name +
+                                                            "\nlast_name: " + patwa.last_name +
+                                                            "\ndob: " + patient.dob +
+                                                            "\ngender: " + patient.sex +
+                                                            "\ntelephone: " + patient.phone_number +
+                                                            "\nheight: " + patient.height_cm +
+                                                            "\nweight: " + patient.weight_kg +  
+                                                            "\nblood_type: " + patient.blood_type;
```

Comment:
You can just use indexed object in your statements (`patients[i].dataValues.<fieldname>`)